.github/workflows/main.yml

```diff
@@ -42,6 +41,43 @@ jobs:
       - name: Lint Client Rust
         run: pnpm clients:rust:lint
 
+  format_and_lint_interface:
+    name: Format & Lint Interface
+    runs-on: ubuntu-latest
+    steps:
+      - name: Git Checkout
+        uses: actions/checkout@v4
+
+      - name: Setup Environment
+        uses: ./.github/actions/setup
+        with:
+          clippy: true
+          rustfmt: true
+
+      - name: Format Interface
+        run: pnpm interface:format
+
+      - name: Lint Interface
+        run: pnpm interface:lint
+
+  build_interface:
+    name: Build Interface
+    runs-on: ubuntu-latest
+    needs: format_and_lint_interface
+    steps:
+      - name: Git Checkout
+        uses: actions/checkout@v4
+
+      - name: Setup Environment
+        uses: ./.github/actions/setup
+        with:
+          rust-toolchain: true
+          cargo-cache-key: cargo-interface
+          solana: true
```

Comment:
I think you can just use the `rustc` that ships with the Solana toolchain, right?
https://github.com/solana-program/address-lookup-table/blob/7562c925f90459ad891d56719c6ea696ae9f555a/.github/workflows/main.yml#L141-L144

Reply:
I need to double check, but I think I was having problems with that since it requires the "nightly" version for the `frozen-abi` feature.

Reply:
Ohh yeah, I think this is why I had to do the speciailization `cfg` annotations in the ALT program.